apps/src/code-studio/pd/foorm/library_editor/FoormLibraryEditorPreview.jsx

```diff
@@ -15,27 +15,33 @@ const styles = {
   }
 };
 
-class FoormEditorPreview extends Component {
```

Comment:
For this component, I think you can use more generic names to de-dupe most of it, and maybe have the parent component pass in a bool to indicate whether you are previewing a library or a form; that way you know whether you want to call `wrapLibraryQuestionPreviewQuestion` and what language to use in the error message.